### PR TITLE
Prop name with the asterisk sets ptopType as required

### PR DESCRIPTION
**Pull request by shapkarin**
Fixes issue #57 

Changes:  If user define prop name with * at the end set propType as required (name* or name:type*)

### CODE REVIEW

File: bin/initialize.js [hunk 1]
```diff
@@ -147,21 +147,6 @@ program
 
 				let numberOfPropTypes = 0;
 
-				let inputPropTypeName = {
```
Comment (shapkarin) [inline bin/initialize.js]: defined but never used

File: bin/initialize.js [hunk 2]
```diff
@@ -193,7 +178,7 @@ program
 						{
 							type: "input",
 							name: "propNames",
-							message: "Prop names",
+							message: 'Prop names (place "*" at the end each name that required)',
```
Comment (rajikaimal) [inline bin/initialize.js]: Let's format the code with Prettier

**Comment by rajikaimal** [comment]
Just run Prettier on all files. Otherwise LGTM!

File: bin/initialize.js (hunk 2 above)
Comment (shapkarin) [reply bin/initialize.js]: OK

**Pull request merged**